themes/bootstrap3/js/common.js

```diff
@@ -349,14 +349,19 @@ var VuFind = (function VuFind() {
       }
     });
 
+    let scriptElement = elm;
+
     if (property === 'innerHTML') {
       elm.replaceChildren(...tmpDiv.childNodes);
     } else if (property === 'outerHTML') {
+      scriptElement = elm.parentNode;
       elm.replaceWith(...tmpDiv.childNodes);
     }
 
-    // Set any attributes (N.B. has to be done before scripts in case they rely on the attributes):
-    Object.entries(attrs).forEach(([attr, value]) => elm.setAttribute(attr, value));
+    if (property !== 'outerHTML') {
+      // Set any attributes (N.B. has to be done before scripts in case they rely on the attributes):
+      Object.entries(attrs).forEach(([attr, value]) => elm.setAttribute(attr, value));
+    }
```

Comment:
Should we perhaps add something like:

```suggestion
    } else {
        console.error("Incompatible parameters passed to setElementContents().");
    }
```

...just so there's a bit of feedback if somebody attempts something illegal?

Reply:
Yes, but only when the 'attrs' is actually not empty. I added it.

Reply:
Yes, good point -- thanks!